examples/raw_cloudevent/build.yaml

```diff
@@ -0,0 +1,8 @@
+targets:
+  $default:
+    builders:
+      source_gen|combining_builder:
+        options:
+          ignore_for_file:
+          - prefer_expression_function_bodies
+
```

Comment:
one extra new-line?

Reply:
Oh! This is only needed if you're using json_serializable. Can remove!

Reply:
ah, gotcha, thanks!

Reply:
This area is still a bit of an arcane art for me ... I've removed `options` section (I think that's what you're saying)..

Reply:
@kevmoo PTAL now, thanks!

Reply:
ruh roh ... lemme see why build check failed first.

Reply:
Ok, fixed now. PTAL when you have a moment, thanks. I will add Cloud Run docs for this after we're able to publish v0.3.0 and remove the pub overrides.